From dd2429a0260dcd723004ec41a7f85a45ae8b5015 Mon Sep 17 00:00:00 2001
From: Ran Shalit <rshalit@habana.ai>
Date: Tue, 25 Feb 2025 14:08:54 +0200
Subject: [PATCH] [FSW-8421] fs1: palad_bloader: fix hw_mem_copy empty func

emb_common_lib was missing in palad_bloader, so hw_mem_copy
was empty waek function (with warning) - and preboot
was not loaded, fixed by this patch

Change-Id: I1f99a34167df9ea162bed2dd683bf85745ba4b1e
Signed-off-by: Ran Shalit <rshalit@habana.ai>
---

diff --git a/arch/fs1/arc/Makefile b/arch/fs1/arc/Makefile
index f6c7fda..d9b41c6 100644
--- a/arch/fs1/arc/Makefile
+++ b/arch/fs1/arc/Makefile
@@ -15,6 +15,9 @@
 PPBOOT_SRCS_C += $(HL_DEV_ARCH)/pll.c
 PPBOOT_SRCS_C += $(shell ls $(EMB_COMMON_MOD_DIR)/arch/$(BASE_DEV_NAME)/common.c)
 
+PALAD_BLOADER_SRCS_C += $(shell ls $(EMB_COMMON_MOD_DIR)/arch/$(BASE_DEV_NAME)/$(HL_TARGET).c)
+PALAD_BLOADER_SRCS_C += $(shell ls $(EMB_COMMON_MOD_DIR)/arch/$(BASE_DEV_NAME)/common.c)
+
 # TODO: uncomment when bootrom exports is available
 #ifeq ($(HL_TARGET),onix)
 #	LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports_onix.elf
